Второй зачет/Зачет 2 задача 2/stack.c

```diff
@@ -0,0 +1,86 @@
+#include "stack.h"
+
+typedef struct Stack
+{
+    int value;
+    Stack* next;
+} Stack;
+
+typedef struct Queue
+{
+    Stack* first;
+    Stack* second;
+} Queue;
+
+Stack* createStack()
+{
+    return (Stack*)calloc(1, sizeof(Stack));
+}
+
+Queue* createQueue()
+{
+    Queue* newQueue = (Queue*)calloc(1, sizeof(Queue));
+    newQueue->first = createStack();
+    newQueue->second = createStack();
+    return newQueue;
+}
+
+int push(Queue** queue, int value)
+{
+    Stack* next = createStack();
+    if (next == NULL)
+    {
+        return 1;
+    }
+
+    next->value = value;
+    next->next = (*queue)->first;
+    (*queue)->first = next;
+    return 0;
+}
+
+int pop(Stack** head)
+{
+    if ((*head) == NULL)
+    {
+        return NULL;
```

Comment:
```suggestion
        return 0;
```
Оно же int возвращает. Плюс по-хорошему необходим некий механизм сигнализации об ошибках.